docs/8-call-status.md

```diff
@@ -0,0 +1,75 @@
+# Nexmo In-App Call States 
+
+In this quick start we are going to explain call statuses for _outgoing calls_ like 
```

Comment:
i think this entire paragraph is redundant.
people will filter what they want to read straight from the titles.

Reply:
it is also call states for ALL calls not outgoing or anything